packages/core/src/browser/menu/context-menu-context.ts

```diff
@@ -0,0 +1,43 @@
+/********************************************************************************
+ * Copyright (C) 2019 TypeFox and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { injectable } from 'inversify';
+import { OS } from '../../common/os';
+
+@injectable()
+export class ContextMenuContext {
+
+    protected _altPressed = false;
+    get altPressed(): boolean {
+        return this._altPressed;
+    }
+
+    protected setAltPressed(altPressed: boolean): void {
+        if (this._altPressed !== altPressed) {
```

Comment:
[minor]: This `if` condition does not make any difference. I propose removing it.

Reply:
👍 it is leftover, i used to have an event in the first attempt